server/middleware/validation/addNewHouse.js

```diff
@@ -18,7 +30,20 @@ const addHouseValidation = async (req, res, next) => {
       image: string().required().url(),
     });
 
-    await houseSchema.validate(req.body, { abortEarly: false });
+    await houseSchema.validate(
+      {
+        title,
+        description,
+        location,
+        rooms,
+        bathrooms,
+        category,
+        price,
+        area,
+        image,
+      },
```

Comment:
after district, you pass the same object properties, just pass ``` req.body ``` instead of making distract